src/components/MaterialUI/Inputs/InputBase.test.js

```diff
@@ -185,4 +185,23 @@ describe("InputBase Component", () => {
 
 		expect(mountedComponent.containsMatchingElement(expected), "to be truthy");
 	});
+
+	it("Renders InputBase with autoComplete", () => {
```

Comment:
Add another test with autoComplete having another value.

Reply:
Sure, will do that now.